def handler(event, _):
    raise Exception('My test error')
